tests/backup/backup_helper.go

```diff
@@ -3445,9 +3447,56 @@ func DeleteAppNamespace(namespace string) error {
 		return "", false, nil
 	}
 	_, err = task.DoRetryWithTimeout(namespaceDeleteCheck, namespaceDeleteTimeout, jobDeleteRetryTime)
+	if err != nil {
+		ns, err = k8sCore.GetNamespace(namespace)
+		if err != nil {
+			return err
```

Comment:
How about we disregard the error message indicating that the namespace does not exist?

Reply:
You are assuming that the only error that can occur is "namespace not found". But if it is failing because of some network connectivity issue, we need to catch that.

Reply:
I mean if err contains "namespace not found" return nil else return err

Reply:
If you see, we come to this stage if and only if the namespace has not been deleted even after issuing a normal delete (See the lines above this code). That means the namespace will still be there and because of some reason it is still not deleted. So we will not get "namespace not found" error here.

Reply:
I hope so. I think err returned by DoRetryWithTimeout is not logged.